Bottom: 7df9b83a2da52497d04fff46a3bc3a34e119497d
Top:    6043a585185a2442d61a448420c146fef3601492
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-22 02:15:39 +0100

Refresh of try-something-with-the-rtrees

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 436ecb6..adc2461 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1244,9 +1244,8 @@ InsertHoles (jmp_buf * e, POLYAREA * dest, PLINE ** src)
       else
 	{
 	  /* link at front of hole list */
-	  tmp = container->next;
+	  curh->next = container->next;
 	  container->next = curh;
-	  curh->next = tmp;
 
           /* Search for which POLYAREA the containing contour belongs to */
           /* FIXME: Perhaps store this information in the heap structure? */
